lib/schedule.js

```diff
@@ -362,6 +393,29 @@ function recurMatch(val, matcher) {
   return false;
 }
 
+function recurValidateProperty(val, min, max) {
+  if (val === null) {
+    return true;
+  }
+
+  // wrap in array
+  if (!(val instanceof Array)) {
+    val = [val];
+  }
+
+  for (var i = 0; i < val.length; i++) {
+    if (typeof val[i] !== 'number' && !(typeof val[i] === 'object' && val[i] instanceof Range)) {
+      return false;
+    } else if (typeof val[i] === 'number' && (val[i] < min || val[i] > max)) {
+      return false;
+    } else if (typeof val[i] === 'object' && val[i] instanceof Range && (val[i].start < min || val[i].end > max)) {
+      return false;
+    }
```

Comment:
@tejasmanohar 
- Never a bad idea. 
- No, you are right. I honestly just lifted it from elsewhere in schedule.js. I'll remove and update the other places it's doing it in another pull request.